core/api/features/steps/Common.py

```diff
@@ -28,12 +29,12 @@ def step_impl(context, name_object):
 
 @step('Validates schema with "{schema}"')
 def step_impl(context, schema):
-    expected = SchemaValidator.validate(context.json_response.json(), schema)
-    assert True is expected
+    result = SchemaValidator.validate(context.json_response.json(), schema)
+    assert True is result
 
 
-@step("Validates response body")
-def step_impl(context):
-    """
-    :type context: behave.runner.Context
-    """
+@step('Validates response body with "{expected_body}"')
+def step_impl(context, expected_body):
+    body_validator = BodyValidator()
+    result = body_validator.validate(context.json_response, expected_body, context.data_table)
+    assert True is result
```

Comment:
improve this assert

Reply:
It works of that way, because it doesn't be a method of a Library